src/Tickets/Commerce/Gateways/PayPal/REST/Create_Order.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\PayPal\REST;
+
+use TEC\Tickets\Commerce\Gateways\PayPal\Repositories\Order;
+
+/**
+ * Class Create_Order
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\PayPal\REST
+ */
+class Create_Order {
+	/**
+	 * Create order.
+	 *
+	 * @since 5.1.6
+	 */
+	public function handle(  ) {
+		$postData = give_clean( $_POST );
+		$formId   = absint( tribe_get_request_var( 'give-form-id' ) );
+
+		$data = [
+			'formId'              => $formId,
+			'formTitle'           => give_payment_gateway_item_title( [ 'post_data' => $postData ], 127 ),
+			'paymentAmount'       => isset( $postData['give-amount'] ) ? (float) apply_filters( 'give_payment_total', give_maybe_sanitize_amount( $postData['give-amount'], [ 'currency' => give_get_currency( $formId ) ] ) ) : '0.00',
```

Comment:
Consider moving this to its own if/set structure so it is readable and the filter can be commented.

Reply:
Additionally, the filter name needs to change.

Reply:
We will, I've added the proper todos to make sure that happens.